### PR TITLE
Small changes in the getting started tutorial

### PR DESCRIPTION
**Pull request by vivanwin**
### PR Description

I have updated the getting started tutorial to use the generative functions and remove some parts of the code that where not needed.

### Some notes

- [ ] Please make sure that you placed the files in an appropriate folder
- [ ] And that the files have indicative names.

- [ ] Please note that Classiq runs automatic code linting, which may minorly alter some files.
  - [ ] If you're familiar with `pre-commit`, you may run `pre-commit install`, and then at each commit, your files will be altered in a similar way

### CODE REVIEW

**Comment by NadavClassiq** [comment]
Hello @vivanwin!
Thank you for this PR. We will review it in the upcoming days and get back to you.

File: tutorials/getting_started/part3_deutsch_jozsa.ipynb [hunk 1]
```diff
@@ -71,22 +70,14 @@
...
-   ],
+   "outputs": [],
    "source": [
     "from classiq import *\n",
     "\n",
     "\n",
     "@qfunc\n",
-    "def my_black_box_prediate(aux: QBit, target: QNum):\n",
-    "    aux ^= target > 30\n",
+    "def my_black_box_prediate(aux: QNum, target: QNum):\n",
```
Comment (matanvax2) [inline tutorials/getting_started/part3_deutsch_jozsa.ipynb:79]: Why was this changed? I think a variable of type `QBit` actually makes more sense in storing the result of a boolean expression..

**Review by matanvax2 — COMMENTED** [review]
I wasn't tasked for this CR, but still wanted to share a couple of comments

**Review by NadavClassiq — APPROVED** [review]
